### PR TITLE
feat: add supports for image with link

### LINKED ISSUE #4406: 升级到2.8后默认编辑器无法为图片增加超链接
### What is version of Halo has the issue?

2.8

### What database are you using?

MySQL 8.x

### What is your deployment method?

Docker

### Your site address.

_No response_

### What happened?

升级到2.8后默认编辑器无法为图片增加超链接。只有修改图片的链接，无法增加超链接。

### Relevant log output

_No response_

### Additional information

是否可以为图片增加-修改alt属性，方便seo优化

### PR DESCRIPTION
**Pull request by hilaliu**
为图片添加外部a标签，支持图片跳转。
fixes https://github.com/halo-dev/halo/issues/4406

```release-note
支持为图片添加跳转链接
```

### CODE REVIEW

File: packages/editor/src/locales/zh-CN.yaml [hunk 1]
```diff
@@ -51,6 +51,7 @@ editor:
       restore_size: 恢复原始尺寸
       edit_link: 修改链接
       edit_alt: 修改图片 alt 属性
+      edit_href: 修改外部 href 属性
```
Comment (ruibaby) [inline packages/editor/src/locales/zh-CN.yaml]:
```suggestion
      edit_href: 修改图片跳转链接
```

File: packages/editor/src/locales/zh-CN.yaml [hunk 2]
```diff
@@ -102,6 +103,7 @@ editor:
     placeholder:
       link_input: 输入链接，按回车确定
       alt_input: 输入图片 alt 属性值，按回车确认
+      alt_href: 输入图片外部链接 href 属性值，按回车确认
```
Comment (ruibaby) [inline packages/editor/src/locales/zh-CN.yaml]:
```suggestion
      alt_href: 输入图片跳转链接，按回车确认
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Comment by ruibaby** [comment]
@GodlessLiu 需要处理一下冲突。

File: packages/editor/src/locales/en.yaml [hunk 3]
```diff
@@ -51,6 +51,7 @@ editor:
       restore_size: Restore original size
       edit_link: Edit link
       edit_alt: Edit alt
+      edit_href: Edit href
```
Comment (ruibaby) [inline packages/editor/src/locales/en.yaml]:
```suggestion
      edit_href: Edit the image hyperlink
```

File: packages/editor/src/locales/en.yaml [hunk 4]
```diff
@@ -102,6 +103,7 @@ editor:
     placeholder:
       link_input: Enter the link and press enter to confirm.
       alt_input: Enter the image alt and press enter to confirm.
+      alt_href: Enter the image href and press enter to confirm.
```
Comment (ruibaby) [inline packages/editor/src/locales/en.yaml]:
```suggestion
      alt_href: Enter the image hyperlink and press enter to confirm.
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Review by JohnNiang — APPROVED** [review]
It works very well.

image

/lgtm

**Pull request merged**